### DISCUSSION TITLE
Balancer

### DISCUSSION THREAD

**Discussion by brunopiras** [syssi/esphome-jbd-bms#68] [Q&A] [answered]
Hi, I have a balancer from Daly, do you think this code will work?

**Comment by syssi** [comment]
Could you provide some details about your model / device?

**Reply by brunopiras** [reply]
Hi,

of course:

Daly electric vehicle lifepo4 battery bms 4S 8S 16s 1A Smart active balancer


https://cdn.globalso.com/dalybms/DALY-Smart-Active-Equalizer-Eq1.jpg

**Reply by syssi** [reply, marked as answer]
I guess this project does support the mentioned device: https://github.com/syssi/esphome-daly-bms
